run-in-docker.sh

```diff
@@ -2,28 +2,32 @@
 
 # Synopsis:
 # Test runner for run.sh in a docker container
-# Takes the same arguments as run.sh (EXCEPT THAT SOLUTION PATH IS RELATIVE)
+# Takes the same arguments as run.sh (EXCEPT THAT SOLUTION AND OUTPUT PATH ARE RELATIVE)
 # Builds the Dockerfile
 # Runs the docker image passing along the initial arguments
 
 # Arguments:
 # $1: exercise slug
-# $2: **RELATIVE** path to solution folder (without trailing slash)
+# $2: **RELATIVE** path to solution folder (with trailing slash)
+# $3: **RELATIVE** path to output directory (with trailing slash)
 
 # Output:
-# [For now] writes the tests output to the terminal
+# Writes a results.json file with the test results in the output directory
 
 # Example:
-# ./run-in-docker.sh two-fer ./relative/path/to/two-fer/solution/folder
+# ./run-in-docker.sh two-fer ./relative/path/to/two-fer/solution/folder/ ./relative/path/to/output/directory/
 
 # If arguments not provided, print usage and exit
-if [ -z "$1" ] || [ -z "$2" ]; then
-    echo "usage: run-in-docker.sh exercise-slug ./relative/path/to/solution/folder"
+if [ -z "$1" ] || [ -z "$2" ] || [ -z "$3" ]; then
+    echo "usage: run-in-docker.sh exercise-slug ./relative/path/to/solution/folder/ ./relative/path/to/output/directory/"
     exit 1
 fi
 
 # build docker image
 docker build -t python-test-runner .
 
 # run image passing the arguments
-docker run -v $PWD/$2:/solution python-test-runner $1 /solution
+docker run \
+    --mount type=bind,src=$PWD/$2,dst=/solution \
+    --mount type=bind,src=$PWD/$3,dst=/output \
```

Comment:
I'm fairly new to Docker. Why not just `--volume $PWD/$2:/solution $PWD/$3:/output`?

Reply:
I'm new to Docker, too. Going through the [docs](https://docs.docker.com/engine/reference/commandline/run/#add-bind-mounts-or-volumes-using-the---mount-flag) I read that `--mount` is recommended over `-v` or `--volume`.
I also found the new syntax more readable.